controller/src/beerocks/master/db/db.cpp

```diff
@@ -3600,3 +3600,20 @@ void db::clear_vap_list()
 }
 
 const std::shared_ptr<db::vaps_list_t> db::get_vap_list() { return m_vap_list; }
+
+bool db::is_prplmesh(const std::string &mac)
```

Comment:
I *don't* want new interfaces that use strings for MAC addresses.

Reply:
Ok.